README.md

```diff
@@ -1,1110 +1,18 @@
-
-There is a utility bundled under /util/mkpasswd/mkpasswd. By default with no arguments it will generate a secure password and the associated hash. This can be used for a password or a token in the configuration. If you already have a password selected, you can supply that on stdin with the -p flag.
-
-```bash
-~/go/src/github.com/nats-io/nats-server/util/mkpasswd > ./mkpasswd
-pass: #IclkRPHUpsTmACWzmIGXr
-bcrypt hash: $2a$11$3kIDaCxw.Glsl1.u5nKa6eUnNDLV5HV9tIuUp7EHhMt6Nm9myW1aS
-```
-
-Add into the server configuration file's authorization section.
-```
-  authorization {
-    user: derek
-    password: $2a$11$3kIDaCxw.Glsl1.u5nKa6eUnNDLV5HV9tIuUp7EHhMt6Nm9myW1aS
-  }
-```
-
-## Monitoring
-
-If the monitoring port is enabled, the NATS server runs a lightweight HTTP server that has the following endpoints: /varz, /connz, /routez, and /subsz. All endpoints return a JSON object. See [NATS Server monitoring](https://nats.io/documentation/managing_the_server/monitoring/) for endpoint examples.
-
-To see a demonstration of NATS monitoring, run a command similar to the following for each desired endpoint:
-
-```
-curl demo.nats.io:8222/varz
-```
-
-To enable the monitoring server, start the NATS server with the monitoring flag `-m` (or `-ms`) and specify the monitoring port.
-
-Monitoring options
-
-    -m, --http_port PORT             HTTP PORT for monitoring
-    -ms,--https_port PORT            Use HTTPS PORT for monitoring (requires TLS cert and key)
-
-To enable monitoring via the configuration file, use `host:port` (there is no explicit configuration flag for the monitoring interface).
+[![License][License-Image]][License-Url] [![FOSSA Status][Fossa-Image]][Fossa-Url] [![ReportCard][ReportCard-Image]][ReportCard-Url] [![Build][Build-Status-Image]][Build-Status-Url] [![Release][Release-Image]][Release-Url] [![Coverage][Coverage-Image]][Coverage-Url] [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1895/badge)](https://bestpractices.coreinfrastructure.org/projects/1895)
 
-For example, running the `nats-server -m 8222` command, you should see that the NATS server starts with the HTTP monitoring port enabled. To view the monitoring home page, go to <a href="http://localhost:8222/" target="_blank">http://localhost:8222/</a>.
+## Documentation
 
-```
-[70450] 2018/08/29 12:48:30.819682 [INF] Starting nats-server version 1.3.0
-[70450] 2018/08/29 12:48:30.819750 [INF] Git commit [not set]
-[70450] 2018/08/29 12:48:30.819918 [INF] Starting http monitor on 0.0.0.0:8222
-[70450] 2018/08/29 12:48:30.819960 [INF] Listening for client connections on 0.0.0.0:4222
-[70450] 2018/08/29 12:48:30.819964 [INF] Server is ready
-```
+* [Official documentation](https://nats.io/documentation)
+* [FAQ](https://nats.io/documentation/faq/)
```

Comment:
Our docs will now be at https://nats-io.github.io/docs with the FAQ here https://nats-io.github.io/docs/faq.html

Reply:
thanks, ok updated